docs/docs/using-the-template.md

```diff
@@ -0,0 +1,158 @@
+# Using the template
+
+Ok, so you've created your project. You've [read the opinions section](opinions.md). You're ready to start doing some work.
+
+Here's a quick guide of the kinds of things we do once our project is ready to go.
+
+## Set up version control
+
+Once you `cd` into the directory created by `ccds` you're ready to go.
+
+Often, we start by initializing a `git` repository to track the code we write in version control and collaborate with teammates. At the command line, you can do this with the following commands which do the following: turn the folder into a git repository, add all of the files and folders created by CCDS into source control (except for what is in the `.gitignore` file), and then make a commit to the repository.
+
+```bash
+git init
+git add .
+git commit -m "CCDS defaults"
+```
+
+We usually commit the entire default CCDS structure so it is easy to track in verison history the changes we make to the structure separately.
+
+Now that default layout is committed, you should push it to a shared repository. You can do this through the interface of whatever source control platform you use. This may be GitHub, GitLab, Bitbucket, or something else.
+
+If you use GitHub and have the [gh cli tool](https://cli.github.com/) you can easily create a new repository for the project from the commandline.
+
+```bash
+gh repo create
+```
+
+You'll be asked a series of questions to set up the repository on GitHub, for example:
+
+```
+gh repo create
+? What would you like to do? Push an existing local repository to GitHub
+? Path to local repository: .  # CCDS generated folder you are in
+? Repository name: testccds    # Name of the project 
+? Repository owner: pjbull     # Github username
+? Description: Testing         # A short description of the project
+? Visibility: Private          # 
+✓ Created repository pjbull/testccds on GitHub
+? Add a remote? Yes
+? What should the new remote be called? origin
+\✓ Added remote https://github.com/pjbull/testccds.git
+? Would you like to push commits from the current branch to "origin"? Yes
+Enumerating objects: 23, done.
+Counting objects: 100% (23/23), done.
+Delta compression using up to 12 threads
+Compressing objects: 100% (20/20), done.
+Writing objects: 100% (23/23), 6.47 KiB | 6.47 MiB/s, done.
+Total 23 (delta 0), reused 0 (delta 0), pack-reused 0
+To https://github.com/pjbull/testccds.git
+ * [new branch]      HEAD -> main
+branch 'main' set up to track 'origin/main'.
+✓ Pushed commits to https://github.com/pjbull/testccds.git
+```
```

Comment:
I feel as though this is a little too much to provide here. It might be better if we just linked out to the gh cli docs.